### ISSUE TITLE
Standardising tournament terminology

### ISSUE THREAD

**Issue by Walavouchey** [ppy/osu-wiki#7401]
I'd appreciate if some tournament-related people could chime in with their opinions

## Game modifiers

After #6920, it seems newer World Cup articles have been starting to use spaces in mod names [as the ASC defines them](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#game-modifiers) (e.g. *Double Time* instead of *Double Time* and *No Fail* instead of *NoFail*), so I suppose those should just be corrected across the wiki, continuing where the PR left off.

I propose using *No Mod* and *Free Mod* instead of *NoMod* and *FreeMod*. Apparently the reason for defining them without spaces in the ASC was due to them being [commonly spelt that way when the ASC was written](https://github.com/ppy/osu-wiki/pull/6920#issuecomment-1049238813). With something like *Mixed Mod* also being put in the [CWC 2020 mappools](https://osu.ppy.sh/wiki/en/Tournaments/CWC/2022#mappools) (with spaces), I'm pretty sure the ASC is just outdated in this regard.

## Tournament stages

These are usually always capitalised in all cases:

- Grand Finals
- Finals
- Semifinals
- Quarterfinals
- Round of N

"Qualifiers" is sometimes written in a mix of uppercase and lowercase, especially in cases like "qualifier maps" or "qualifier mappool". To be consistent with other stage names, it should always be capitalised.[^beatmap-category-note]

[^beatmap-category-note]: As a side note, it's almost the same as the issue of when to capitalise beatmap categories, i.e. writing "Ranked beatmap" or "ranked beatmap", although the difference here is that "ranked", "qualified" and "graveyarded" are used as adjectives, but "qualifier" in "qualifier map" would be a qualifying noun. Personally I think they should be in lowercase when used as verbs, but otherwise it could go either way.

When referring to other stages, we could either only capitalise the stage name, i.e. "Regular stage" or "Swiss stage" (you'd get "Regular stage mappool" etc.), or capitalise "stage" as well ("Regular Stage mappool"). I've thought otherwise before, but the latter might be more consistent if the idea is to always capitalise stages, in terms of treating them as whole units. If we do that though, then I'd honestly prefer always using "Qualifiers" over "Qualifier Stage" for the same reason "Finals Stage" might look a little off.

---

*Phases* are a bit different from stages, in that they are not part of tournament brackets. I'd propose not to capitalise a phase, so in sentences you'd get "registration phase", "screening phase", "judging phase" (mainly in contests), etc.

## Other

- *mappool* instead of *pool* (for formality, similar to *beatmap* instead of *map*)
- *best-of-N* instead of *best of N* ([going by Wikipedia](https://en.wikipedia.org/wiki/Playoff_format#Best-of_formats)), although "first to N" can be with spaces. not a proper noun ("a best-of-17 match", not "a Best-of-17 match").
- *double-elimination* when it modifies another noun (i.e. "double-elimination stage/format"), not capitalised ([again going by Wikipedia](https://en.wikipedia.org/wiki/Double-elimination_tournament))
- I would love for someone to explain why "Tournament Management" is capitalised everywhere (469 vs 112 instances)

---

Let me know if I missed anything else that's inconsistent in tournament articles

**Comment by LeoFLT** [comment]
Looks good to me, most of these are the de facto standard already.

> I would love for someone to explain why "Tournament Management" is capitalised everywhere (469 vs 112 instances)

That wording in specific is ancient terminology, nobody really knows at this point (3 different people worked on the wiki articles at the very least, things tend to get messy). Would be glad to change it to something that's more consistent with the rest of the article.

**Comment by Walavouchey** [comment]
Another term I forgot about, should it be **Losers Finals**, **Losers' Finals** or **Loser's Finals** (and winners respectively)?

**Comment by cl8n** [comment]
the mod names seem pretty weird to me, in my time playing osu I've practically only seem them written pascal case in tournament contexts, especially so for NoMod/FreeMod/ForceMod and friends. I think that's also how the game prints them when managing a lobby?

**Comment by Walavouchey** [comment]
> I think that's also how the game prints them when managing a lobby?

not sure how relevant that would be. the button to enable FM in a multiplayer lobby is called "Free Mods" and in the !mp command is `mp mods Freemod`

> though in the article you point out "No Mod" probably would be better since the mods are written like that already. I think they should just be written to fit in and not standardized across the whole wiki.

you mean writing something like "using no mods" instead of "using No Mod" in writing? sure, but i'm talking about the term as it's used in tournament mappools and otherwise, since fitting it in with how the rest of the mods are written (with a space) would make these terms more consistent.

and if we're changing this then i'm not sure why it would make sense not to change it across the whole wiki

**Comment by cl8n** [comment]
> you mean writing something like "using no mods" instead of "using No Mod" in writing? sure, but i'm talking about the term as it's used in tournament mappools and otherwise, since fitting it in with how the rest of the mods are written (with a space) would make these terms more consistent.

no, I'm talking about the mappools too, in that CWC article you linked the mods are like "Hard Rock" etc so I think it would make sense to write "No Mod" "Free Mod" there. but if tournament is writing HardRock etc then it'd make sense to match accordingly
